src/components/widgets/AssocListPopup/AssocListPopup.tsx

```diff
@@ -27,7 +27,8 @@ export interface IAssocListRecord {
 export interface IAssocListActions {
     onSave: (bcNames: string[]) => void,
     onFilter: (bcName: string, filter: BcFilter) => void,
-    onDeleteTag: (bcName: string, dataItem: DataItem) => void,
+    onDeleteTag: (bcName: string, depth: number, widgetName: string,
```

Comment:
1) Adding new arguments brokes the contract for this interface
2) You don't really need two arguments that you've add: `depth` is just an alias for `dataItem.level` and `dataItem` is already an argument, and you can get `widgetName` in epic from the `store.view.popupData`